arkouda/pdarrayclass.py

```diff
@@ -1217,6 +1217,35 @@ def list_registry() -> List[str]:
 
     return registered_list
 
+def list_symbol_table() -> List[str]:
+    """
+    Return a list containing the names of all objects in the symbol table
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the symbol table
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    symbol_list: List[str] = []
+
+    if info(AllSymbols) not in [EmptyRegistry, EmptySymbolTable]:
```

Comment:
Since it's a network call, It's probably a good idea to store the result of `info(AllSymbols)` so you don't need to call it again in the for loop below.